docs/content/sidebars/references.js

```diff
@@ -69,17 +69,27 @@ const references = [
 			id: 'references/sui-sdks',
 		},
 		items: [
+			'references/rust-sdk',
 			{
 				type: 'link',
 				label: 'Sui TypeScript SDK',
 				href: 'https://sdk.mystenlabs.com/typescript',
 			},
+      {
+				type: 'link',
```

Comment:
nit: alphabetize perhaps?
dApp
Go
Py
rust
ts

Reply:
The parent page would need to do this, too

Reply:
Makes sense